src/bma400.rs

```diff
@@ -9,6 +9,9 @@ pub enum IntType {
     Inactivity,
 }
 
+const ACTIVITY_DURATION: u16 = 15;
```

Comment:
Probably better to have these as parameters to `configure_interrupt(..., activity_duration, inactivity_duration)`